cmd/pxe.c

```diff
@@ -1709,7 +1759,11 @@ static int do_sysboot(cmd_tbl_t *cmdtp, int flag, int argc, char * const argv[])
 	if (prompt)
 		cfg->prompt = 1;
 
+#ifdef CONFIG_BOOTMENU
```

Comment:
Shouldn't here be #ifndef ?

Reply:
Thanks, good catch.  Should be #ifndef CONFIG_CMD_BOOTMENU - I had the wrong define as well.

Reply:
Force-pushed with correcting the ifdef, also ifdefing to use the bootmenu when possible when doing true PXE boot.